src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -2585,7 +2585,7 @@ private async Task<(long Index, object Result)> SendToLeaderAsyncInternal(Comman
         private async Task<(long Index, object Result)> SendToLeaderAsyncInternal(TransactionOperationContext context, CommandBase cmd)
         {
             //I think it is reasonable to expect timeout twice of error retry
-            var timeoutTask = TimeoutManager.WaitFor(Engine.OperationTimeout, _shutdownNotification.Token);
+            var timeoutTask = TimeoutManager.WaitFor(TimeSpan.FromMinutes(60), _shutdownNotification.Token);
```

Comment:
Why is that?

Reply:
we try to hunt why we got random timeout failures that cause to 4-5 tests simultaneously to fail

Reply:
Something that @karmeli87 forgot to write here. This PR will not be merged. Just a pure bug hunting code.

@karmeli87 maybe track how much time we wait with stopwatch and if it is > Engine.OperationTimeout do a console writeline with some debug info? Thoughts?